rmw_connext_cpp/src/functions.cpp

```diff
@@ -31,25 +31,25 @@
 #  pragma clang diagnostic ignored "-Wreturn-type-c-linkage"
 # endif
 #endif
-#include <ndds/ndds_cpp.h>
-#include <ndds/ndds_requestreply_cpp.h>
+#include "ndds/ndds_cpp.h"
+#include "ndds/ndds_requestreply_cpp.h"
```

Comment:
For these lines cpplint actually doesn't care. I reverted them to how they were before the PR.